lgtm.yml

```diff
@@ -21,6 +21,8 @@ extraction:
       - libgmock-dev
       - uuid
       - uuid-dev
+      - libyang
+      - libyang-dev
```

Comment:
libyang-dev one package is enough?

Reply:
Fixed, the lgtm seems not run UT so libyang-dev will enough.